libs/shared/advert-info/src/lib/shared-advert-info.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import Chip from '@material-ui/core/Chip';
+
+import './shared-advert-info.scss';
+
+interface Category {
+  title: string;
+  value: string;
+}
+
+/* eslint-disable-next-line */
+export interface SharedAdvertInfoProps {
+  title: string;
+  description: string;
+  creator: string;
+  image?: string;
+  categories?: Category[];
+  onGetStartedClick: () => void;
+}
+
+export const SharedAdvertInfo = (props: SharedAdvertInfoProps) => {
+  return (
+    <div className="main-part">
+      <img
+        className="bg-image"
+        alt={props.title}
+        src={
+          props.image
+            ? props.image
+            : 'https://images.ctfassets.net/nm1bt7p4165n/7aK9MEwnD0Y1weylWYzTy7/d2cfb80b084f72d7c6bb275af5423efb/Birstonas2_small.png?w=1920&q=50'
```

Comment:
don't use external link from unapproved source, wee have custom storage for images